tools/novoalign.py

```diff
@@ -177,7 +177,7 @@ def align_one_rg_bam(self, inBam, refFasta, outBam, rgid=None, options=None, min
         tmp_sam = util.file.mkstempfname('.novoalign.sam')
         tmp_sam_err = util.file.mkstempfname('.novoalign.sam.err')
         cmd = [self.install_and_get_path(), '-f', one_rg_inBam] + list(map(str, options))
-        cmd = cmd + ['-F', 'BAMPE', '-d', self._fasta_to_idx_name(refFasta), '-o', 'SAM']
+        cmd = cmd + ['-F', 'BAM', '-d', self._fasta_to_idx_name(refFasta), '-o', 'SAM']
```

Comment:
We should probably have a comment here noting that the use of `-F BAM` is possible via the newer Novoalign version, since it does not seem to be mentioned in the current reference manual.